app/src/main/res/values/strings.xml

```diff
@@ -3,50 +3,53 @@
     <string name="app_name" translatable="false">Seedvault</string>
     <string name="backup">Backup</string>
```

Comment:
As in?

Reply:
This is just a simple string used in many places (such as the app name on CalyxOS), and thus I don't think it needs any more explanation.

A backup app being called "Backup" is as simple as it gets.

Reply:
Until it appears for translation with no context, which is hard to read out of being presented with a verb when it should have been a noun or vice versa when translated.